README.md

```diff
@@ -382,151 +424,454 @@ Jest uses Jasmine 2 by default. An introduction to Jasmine 2 can be found
+
+It's easier to understand this with an example. Let's say you have a method `bestLaCroixFlavor()` which is supposed to return the string `'grapefruit'`.
+Here's how you would test that:
 
 ```js
-  const mockFn = jest.fn();
-  mockFn();
-  expect(mockFn).toBeCalled();
+describe('the best La Croix flavor', () => {
+  it('is grapefruit', () => {
+    expect(bestLaCroixFlavor()).toBe('grapefruit');
+  });
+});
+```
 
-  // With a mock implementation:
-  const returnsTrue = jest.fn(() => true);
-  console.log(returnsTrue()) // true;
+In this case, `toBe` is the matcher function. There are a lot of different matcher functions, documented below, to help you test different things.
+
+The argument to `expect` should be the value that your code produces, and any argument to the matcher should be the correct value. If you mix them up, your tests will still work, but the error messages on failing tests will look strange.
+
+### `.lastCalledWith(arg1, arg2, ...)`
+
+If you have a mock function, you can use `.lastCalledWith` to test what arguments it was last called with. For example, let's say you have a `applyToAllFlavors(f)` function that applies `f` to a bunch of flavors, and you want to ensure that when you call it, the last flavor it operates on is `'mango'`. You can write:
+
+```js
+describe('applying to all flavors', () => {
+  it('does mango last', () => {
+    let drink = jest.fn();
+    applyToAllFlavors(drink);
+    expect(drink).lastCalledWith('mango');
+  });
+});
 ```
 
-### `jest.isMockFunction(fn)`
-Determines if the given function is a mocked function.
+### `.not`
 
-### `jest.genMockFromModule(moduleName)`
-Given the name of a module, use the automatic mocking system to generate a mocked version of the module for you.
+If you know how to test something, `.not` lets you test its opposite. For example, this code tests that the best La Croix flavor is not coconut:
 
-This is useful when you want to create a [manual mock](https://facebook.github.io/jest/docs/manual-mocks.html) that extends the automatic mock's behavior.
+```js
+describe('the best La Croix flavor', () => {
+  it('is not coconut', () => {
+    expect(bestLaCroixFlavor()).not.toBe('coconut');
+  });
+});
+```
 
-### `jest.mock(moduleName, ?factory)`
-Indicates that the module system should always return a mocked version of the specified module from `require()` (e.g. that it should never return the real module).
+### `.toBe(value)`
+
+`toBe` just checks that a value is what you expect. It uses `===` to check
+strict equality.
+
+For example, this code will validate some properties of the `beverage` object:
 
 ```js
-  jest.mock('moduleName');
+let beverage = {
+  ounces: 12,
+  frenchName: 'pamplemousse',
+};
+
+describe('the can', () => {
+  it('has 12 ounces', () => {
+    expect(can.ounces).toBe(12);
+  });
 
-  const moduleName = require('moduleName'); // moduleName will be explicitly mocked
+  it('has a silly name', () => {
```

Comment:
I know this is all just fun and tests but I think we shouldn't make fun of other cultures or languages :) Worst thing we wanna do is offend people.